src/ApmLogger.php

```diff
@@ -45,7 +45,7 @@ private function getLevelValue(string $level): int
     /**
      * @inheritDoc
      */
-    public function log($level, $message, array $context = array())
+    public function log($level, string|\Stringable $message, array $context = []) : void
```

Comment:
The union type and use of `Stringable` will force the minimum supported version of PHP to PHP 8.0. That will be a major change and require a bit more consideration.